src/guides/guide_changi_short.md

```diff
@@ -24,10 +24,11 @@ Connection data:
 - Network name: **MetaChain**
 > ⚠️ **Warning: Due to potential network overloads, it is advised to use a local node over the public node to connect with Metamask. Follow the guide: [Run a Local Node in Changi](./guide_changi.md).**
 
-- RPC URL: https://changi.dfi.team
-- Chain ID: **1133** (Specific to Changi TestNet)
-- Currency symbol: **DFI**
-- Block explorer URL: https://meta.defiscan.live
+| Network | RPC URL | Chain ID | Currency symbol | Block Explorer (Optional) |
+|---------|---------|----------|----------------|----------------|
+|Changi Testnet| http://changi.dfi.team| 1133 | DFI | https://meta.defiscan.live |
```

Comment:
```suggestion
|Changi Testnet / Managed Node| https://changi.dfi.team | 1133 | DFI | https://meta.defiscan.live |
|Changi Testnet / Local Node | https://127.0.0.1:20551 | 1133 | DFI | https://meta.defiscan.live |
```

Please note that the managed node are public nodes run by the core-dev team for convenience and that they may be susceptible to rate limits.